requirements.txt

```
@@ -1,3 +1,5 @@
 slackclient==1.3.1
 jigsaw==3.2.0
 python-dotenv==0.10.1
+praw==6.1.1
+
```

Add certifi to the requirements
```suggestion
certifi==2019.3.9

```

---

Delegated to @lavenduale